x-pack/legacy/plugins/uptime/server/graphql/unsigned_int_scalar/resolvers.ts

```diff
@@ -4,34 +4,48 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { GraphQLScalarType, Kind } from 'graphql';
+import { GraphQLScalarType, Kind, ValueNode } from 'graphql';
 import { UMServerLibs } from '../../lib/lib';
 import { CreateUMGraphQLResolvers } from '../types';
 
+export const serialize = (value: any): number => {
+  // `parseInt` will yield `2019` for a value such as "2019-07-08T16:59:09.796Z"
+  if (isNaN(Number(value))) {
+    return Date.parse(value).valueOf();
```

Comment:
What does `valueOf()` get you here?

Reply:
Indeed, not needed.

Fixed in https://github.com/elastic/kibana/pull/40526/commits/cf70ed5f33bca94f4d48acbfbe84e488855e0617.